src/manager.js

```diff
@@ -43,7 +43,14 @@ export default class Manager {
                     return fs.readFileAsync(configFilePath);
                 })
                 .then((buffer) => {
-                    this.config = JSON.parse(buffer.toString());
+                    const config = JSON.parse(buffer.toString());
+
+                    // Normalize all paths in the config object
+                    Object.keys(config.projects).forEach(repo => {
```

Comment:
for consistency's sake, do you mind using lodash instead of Object.keys and Array.prototype.forEach ?
respectively _.keys and _.forEach